### PR TITLE
Adding Qwt 6.2.0

### PR DESCRIPTION
**Pull request by kenfred**
Specify library name and version:  **qwt/6.2.0**

Updating recipe to support the latest version of QWT. This is requested in https://github.com/conan-io/conan-center-index/issues/6895.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/qwt/config.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 versions:
   "6.1.6":
     folder: "all"
+  "6.2.0":
+    folder: "all"
```
Comment (ericLemanissier) [inline recipes/qwt/config.yml:5]:
```suggestion
    folder: "all"

```

**Review by uilianries — APPROVED** [review]

**Comment by kenfred** [comment]
These build failures will be resolved when https://github.com/conan-io/conan-center-index/pull/7232 is merged. It should go in first.

**Comment by ericLemanissier** [comment]
You can do a single PR for both changes

**Comment by kenfred** [comment]
> You can do a single PR for both changes

I would have thought that would be discouraged. At this point, would you like me to combine them or see it through with the two PRs?

**Comment by ericLemanissier** [comment]
@kenfred ycan you rebase your branch on top of the current master in order to get the windows fix ?

**Review by ericLemanissier — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
Why hasnt the bot merged this? 😖

**Comment by prince-chrismc** [comment]
@uilianries your approval is not counted by the bots (its behind a merge commit)

**Review by SSE4 — APPROVED** [review]

**Pull request merged**